### PR TITLE
[NO-ISSUE] Adding helm installation in Makefile

### PR DESCRIPTION
**Pull request by gaohoward**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -271,11 +271,15 @@ $(CONTROLLER_GEN): $(LOCALBIN)
 
 .PHONY: envtest
 envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
-$(ENVTEST): $(LOCALBIN)
+$(ENVTEST): $(LOCALBIN) helm
```
Comment (brusdev) [inline Makefile:274]: envtest doesn't depend on helm, maybe helm could just be a prerequisite for test-mk test-mk-v test-mk-do test-mk-do-v test-mk-do-fast test-mk-do-fast-v

Comment (gaohoward) [reply Makefile:274]: +1

**Review by brusdev — APPROVED** [review]

File: controllers/common_util_test.go [hunk 2]
```diff
@@ -72,8 +72,14 @@ var chars = []rune("hgjkmnpqrtvwxyzslbcdaefiou")
 var defaultPassword string = "password"
 var defaultSanDnsNames = []string{"*.apps.artemiscloud.io", "*.tests.artemiscloud.io"}
 var okDefaultPwd = "okdefaultpassword"
+var helmCmd = GetHelmCmd()
 
-const helmCmd = "helm"
+func GetHelmCmd() string {
+	if localHelm, ok := os.LookupEnv("LOCALBIN"); ok {
```
[the remark this answers is not in the document]
Comment (gtully) [reply controllers/common_util_test.go:78]: apologies for being so late to review.

[the remark this answers is not in the document]
Comment (gaohoward) [reply controllers/common_util_test.go:78]: +1 
Thanks Gary. I'll update the PR.

File: Makefile [hunk 3]
```diff
@@ -145,7 +145,7 @@ vet: ## Run go vet against code.
 	go vet -composites=false ./...
 
 ## Run tests.
-test test-v: TEST_VARS = KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" RECONCILE_RESYNC_PERIOD=5s
+test test-v: TEST_VARS = LOCALBIN=$(LOCALBIN) KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" RECONCILE_RESYNC_PERIOD=5s
```
Comment (gtully) [inline Makefile:148]: With the HELM var I think this def of localbin is no longer necessary

Comment (gaohoward) [reply Makefile:148]: Yeah I'll remove it and verify. Thanks!

**Review by gtully — APPROVED** [review]

**Pull request merged**